docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-dynamic-tasks-proc.adoc

```diff
@@ -0,0 +1,53 @@
+[id='case-management-dynamic-tasks-proc']
+= Starting dynamic tasks and processes
+
+Dynamic tasks can be added to a case during runtime. Dynamic tasks are a way to address changing situations, where an unanticipated change during the case requires a new task or process to be incorporated into the case in response. 
+
+Dynamic tasks are added during runtime by way of a case application. For demonstration purposes, the {CENTRAL} distribution includes a Showcase application where you can start a new dynamic task or process for the IT Orders application. For more information about installing and running the Showcase application, see _Showcase application installation_.
+//xreflink TBA
+
+.Prerequisites
+
+* The IT Orders project is deployed in {CENTRAL}.
+* The Showcase application .war file has been deployed.
+* You have deployed an IT Orders case instance in Showcase.
+
+.Procedure 
+. With the IT Orders project deployed and running in {CENTRAL}, in a web browser, navigate to the Showcase login page `http://localhost:8080/rhpam-case-mgmt-showcase/`. 
++
+If you have configured {CENTRAL} to display the *Apps launcher* button, use it to open a new browser window with the Showcase login page.
++
+image::apps-launcher-showcase-button.png[]
++
+For instructions on how to configure the *Apps launcher* button in {CENTRAL} see the _Showcase application installation_ instructions.
+//xreflink TBA
+. Log in to the Showcase application using your {CENTRAL} login credentials.
+. Select an active case instance from the list to open it.
+. Under *Overview* -> *Actions* -> *Available*, click the image:dotdotdotbutton.png[] button next *New user task* or *New process task* to add a new dynamic task.
++
+.Showcase dynamic actions
+image::showcase-dynamic-actions.png[]
+* To create a dynamic user task, start a *New user task* and complete the required information:
++
+image::showcase-dynamic-user-task.png[]
+* To create a dynamic process task, start a *New process task* and complete the required information:
++
+image::showcase-dynamic-process-task.png[]
+. To action a dynamic user task in {CENTRAL}, click *Menu* -> *Track* -> *Task Inbox*. The user task that was added dynamically using the Showcase application appears in the *Task Inbox*.
```

Comment:
You say "to action" a couple of times. Odd to me, but I assume it's just the standard term.

Reply:
Yeah I mean, if there's a task there it's going to require the person it's assigned to to take some kind of action, but that action is going to be different depending on what kind of task it is. I'm always open to suggestions if you have an idea of a way to say it better :)